packages/web-react/.eslintrc.js

```diff
@@ -175,5 +175,11 @@ module.exports = {
         '@typescript-eslint/no-var-requires': 'off',
       },
     },
+    {
+      files: ['tests/**/*.{ts,tsx}'],
+      rules: {
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Please, add a link to the ESLint documentation for this rule, and also provide a descriptive comment, why we are turning this rule off. Thanks.